Froger/MainLoop.java

```diff
@@ -0,0 +1,121 @@
+package Froger;
+
+import javafx.animation.Animation;
+import javafx.animation.KeyFrame;
+import javafx.animation.Timeline;
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Group;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.paint.Color;
+import javafx.scene.paint.ImagePattern;
+import javafx.scene.text.Font;
+import javafx.stage.Stage;
+import javafx.util.Duration;
+
+
+public class MainLoop extends Application {
+
+    private int score = 0;
+    private int sizeMapX = 20;
+    private int sizeMapY = 10;
+    private Group root = new Group();
+    private GridMap gridMap = new GridMap(sizeMapX, sizeMapY);
+    private Frog frog;
+    private Cars cars[];
+    private Gold gold;
+    private Timeline tl;
+    private HealthPoints healthPoints;
+    private Label lbScore;
+
+    @Override
+    public void start(Stage primaryStage) {
+        Scene scene = new Scene(root, 700, 700, Color.WHITE);
```

Comment:
w kodzie masz pelno ``magic numbers`` czyli stalych liczbowych, o ile stringi mozna jeszcze odpuscic to liczbowe wartosci zawsze tworz jako stale bo zapewniam ze jak za 2 miesiace siadziesz do kodu to nie bedziesz pamietac ze te 700 to ``MAX_HEIGHT`` a jakies inne to ``MAX_FPS``